[docs] Mention that `craftcms` is best with Craft CMS 4+ (#4925)
ddev/ddev: docs/content/users/quickstart.md

=== "Craft CMS"

>     ## Craft CMS (4)

Reviewer: I wouldn’t limit this to v4 here because you’re addressing v3 in this new note.

```suggestion
    ## Craft CMS
```

    Start a new [Craft CMS](https://craftcms.com) project or retrofit an existing one.

    !!!tip "Compatibility with Craft CMS 3"

>         The `craftcms` project type is best with Craft CMS 4+. If you are using Craft CMS 3 you may prefer to use the `php` project type. Craft CMS 4+ (and thus DDEV's support) are more opinionated about some settings, and you may want to just manage your own settings. More information is available in [issue](https://github.com/ddev/ddev/issues/4650).

Reviewer: Can whittle this down for efficiency, grammar, and link context. If I’m following correctly, it’d also be good to mention Craft 4, then Craft 3 and the recommended action. As written the tip mentions Craft 3, then the text mentions Craft 4, Craft 3, and Craft 4 again and it’s a bit of a jumble. I hope this is more straightforward and still what you intended.

```suggestion
        The `craftcms` project type is best with Craft CMS 4+, which is more opinionated about some settings. If you are using Craft CMS 3, you may want to use the `php` project type and [manage settings yourself](https://github.com/ddev/ddev/issues/4650).
```
